contentful/export-processor/content-types/section.js

```diff
@@ -152,6 +152,133 @@ export class Section {
     return minimumPathsForRecommendations;
   }
 
+  /**
+   * Calcutes paths to select each answer to each question
+   */
+    getPathsForAllAnswers() {
+        const allAnswerPaths = [];
+
+        // Log ids of all answers in section
+        const allAnswers = this.questions.flatMap(question => question.answers.map(answer => answer.id));
+
+        // Log ids of answers used in minimum paths for recommendations
+        const allRecommendationPaths = this.minimumPathsForRecommendations;
+        const answersUsed = Object.values(allRecommendationPaths).flat().reduce((previous, current) => {
+            if (previous.indexOf(current.answer.id) === -1) {
+                previous.push(current.answer.id);
+            }
+            return previous;
+        }, []);
```

Comment:
Answers should be unique, I think this can be simplified to 
```suggestion
        const answersUsed = Object.values(allRecommendationPaths).flatMap(step=>step.answer.id)
```